execution/src/evm.rs

```diff
@@ -188,7 +188,10 @@ impl<R: ExecutionRpc> EvmState<R> {
                         .await?;
 
                     let storage = self.storage.entry(*address).or_default();
-                    let value = *account.slots.get(&slot_ethers).unwrap();
+                    let slot_ethers_u256 = &slot.as_le_bytes();
```

Comment:
I took another look here, 
specifically, `slot` name is not available, because this is the name of the parameter, and it would not fully shadow it.
It is not fully shadowed in a sense that it is still respected as the original parameter type, and not the casted type.